src/pages/Main/Filter/Filter.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import COLOR_LIST from './colorData';
+import SIZE_LIST from './sizeData';
+import './Filter.scss';
+
+const Filter = () => {
+  const [priceValue, setPriceValue] = useState(50000);
+  const handlePrice = e => {
+    setPriceValue(e.target.value);
+  };
+
+  const [filterData, setFilterData] = useState([]);
+  const filtering = url => {
+    fetch(`http://10.58.52.193:3000/products/all?${url}`, {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json;charset=utf-8' },
+    })
+      .then(response => response.json())
+      .then(res => setFilterData(res));
+    console.log(filtering);
+  };
+
+  return (
+    <div className="filter">
+      <div className="filterInner">
+        {/* gender */}
+        <div className="productGender filterLayout">
+          <h3>성별</h3>
+          <div className="checkBoxList">
+            <input type="checkbox" id="checkWomen" className="checkBoxInput" />
+            <label for="checkWomen" onChange={filtering('gender=2')} />
```

Comment:
```suggestion
            <label for="checkWomen" onChange={() => filtering('gender=2')} />
```
JSX 문법 안에서 함수에 인자를 담아서 호출하게 되면 즉시 실행이 되기 때문에
위와 같은 방식으로 적용해주어야 합니다!